cdap-explore/src/test/java/co/cask/cdap/explore/table/SchemaConverterTest.java

```diff
@@ -210,13 +225,13 @@ public void testUnsupportedTypes() {
   @Test
   public void testSupportedTypes() throws Exception {
     // Should not throw an exception
-    SchemaConverter.toHiveSchema(NotRecursive.class);
+    new SchemaConverter(false).toHiveSchema(NotRecursive.class);
```

Comment:
I think this only tests the case where escaping is on. Should this also test the other case?